homeassistant/components/minecraft_server/sensor.py

```diff
@@ -176,3 +180,19 @@ def __init__(self, server: MinecraftServer) -> None:
     async def async_update(self) -> None:
         """Update maximum number of players."""
         self._state = self._server.players_max
+
+class MinecraftServerMOTDSensor(MinecraftServerSensorEntity):
+    """Representation of a Minecraft Server MOTD sensor."""
+
+    def __init__(self, server: MinecraftServer) -> None:
+        """Initialize MOTD sensor."""
+        super().__init__(
+            server=server,
+            type_name=NAME_MOTD,
+            icon=ICON_MOTD,
+            unit=UNIT_MOTD,
+        )
+
+    async def async_update(self) -> None:
+        """Update MOTD."""
+        self._state = self._server.motd
```

Comment:
The MOTD can potentially be bigger than Home Assistant supports?

Reply:
Can it? What's limit on home assistant's state variable? 59char is Minecraft limit (doesn't account for color markup, which wouldn't make it to home assistant (and didn't on my server here) .

Reply:
Aah that will work indeed 👍